src/main/scala/outwatch/dom/Handlers.scala

```diff
@@ -22,7 +19,7 @@ trait Handlers {
   def createBoolHandler(defaultValues: Boolean*) = createHandler[Boolean](defaultValues: _*)
   def createNumberHandler(defaultValues: Double*) = createHandler[Double](defaultValues: _*)
 
-  def createHandler[T](defaultValues: T*): IO[Handler[T]] = Sink.createHandler[T](defaultValues: _*)
+  def createHandler[T](defaultValues: T*): IO[Handler[T]] = Handler.create[T](defaultValues: _*)
```

Comment:
I know, this was there before. but why have two methods for creating a handler? This is `Handler.create` and `createHandler`.

Reply:
I `Handler.create` because it though the place to create a handler should be in `Handler` (somewhat analogous to `Sink.create`, `Observable.create`...). I kept `Handlers.createHandler` for backwards compatibility, maybe it should be deprecated?

Reply:
Good idea. Deprecating the old one sounds like a good idea. What about the `createNumberHandler`, `createBoolHandler`, ...? Not sure whether, they really add value or whether something like `Handler.create[Boolean]` is good enough.

Reply:
Yes, I think something like `Handler.create[Boolean]` is good enough. The ones that add some value is the ones taking event types, such as `createInputHandler`, `createMouseHandler` as one doesn't need to import the respective events.

Reply:
Maybe add them to `Handler` then? Makes them more consistent.

Reply:
I think prefer `Handler` completely generic. Plus `Handler` is in the `outwatch` package (not in `outwatch.dom` package), so it shouldn't "know"  about dom types.

Reply:
You are right!